homeassistant/components/enphase_envoy/quality_scale.yaml

```diff
@@ -0,0 +1,104 @@
+rules:
+  # Bronze
+  action-setup:
+    status: done
+    comment: |
+      only actions implemented are platform native ones.
+      switch.turn_on, switch.turn_off, switch.toggle,
+      number.set_value
+      select.select.
```

Comment:
Let's not name them explicitly as they can get out of sync

Reply:
removed the names